db/tasks.php

```diff
@@ -0,0 +1,38 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Task definition for tool_mergeusers.
+ * @author    Nicola Vallinoto <n.vallinoto@liguriadigitale.it>
+ * @copyright 2023 Liguria Digitale (https://www.liguriadigitale.it)
+ * @package   tool/mergeusers
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$tasks = array(
+    array(
+        'classname' => 'tool_mergeusers\task\get_queue_merging_requests',
```

Comment:
every time I read the name of the class it rings me a bell as if it was to ask for a list of merge request somewhere.  And this is not the goal of this task, not at all.

So, maybe a different name would be better, like `queue_to_process_pending_merge_requests`.

Reply:
done with the name you suggested